certified-connectors/TxtSync/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2235 @@
+                "From",
+                "Message",
+                "To"
+              ]
+            }
+          }
+        ]
+      }
+    },
+    "/sms/send/bulk": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "string"
+            }
+          }
+        },
+        "operationId": "SendBulkSMS",
+        "summary": "Send Bulk SMS",
+        "description": "Send SMS by List of Numbers or Tag Names",
+        "parameters": [
+          {
+            "name": "x-api-key",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "M3Yf9L0njL8eL9ukphahq9zk5LFRIOvn2fT4BcmV",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "Content-Type:",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/json",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "x-zapier",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "true",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "From": {
+                  "type": "string",
+                  "description": "The Sender ID or E164 number to send from",
+                  "title": "From",
+                  "x-ms-visibility": ""
+                },
+                "To": {
+                  "type": "array",
+                  "items": {
+                    "type": "string",
+                    "description": "List of E164 numbers to send to",
+                    "title": "To",
+                    "x-ms-visibility": ""
+                  },
+                  "description": "To"
+                },
+                "ToTagName": {
+                  "type": "array",
+                  "items": {
+                    "type": "string",
+                    "description": "List of tag names to send to. The tag must exist in TxtSync and have contacts allocated to it",
+                    "title": "To Tag Names",
+                    "x-ms-visibility": ""
+                  },
+                  "description": "ToTagName"
+                },
+                "Message": {
+                  "type": "string",
+                  "description": "The message body of the message",
+                  "title": "Message",
+                  "x-ms-visibility": ""
+                }
+              },
+              "required": [
+                "From",
+                "Message"
+              ]
+            }
+          }
+        ]
+      }
+    },
+    "/system/applications/2/webhooks": {
+      "x-ms-notification-content": {}
```

Comment:
@TxtSync Empty operations can be removed

Reply:
@TxtSync Please remove this block, Please let us know the reason if you want to keep this.